tests/unit/core/api/helpers.js

```diff
@@ -0,0 +1,22 @@
+/* global Response */
+export function generateHeaders(
+  headerData = { 'Content-Type': 'application/json' }
+) {
+  const response = new Response();
+  Object.keys(headerData).forEach((key) => (
+    response.headers.append(key, headerData[key])
+  ));
+  return response.headers;
+}
+
+export function createApiResponse({
```

Comment:
I think this could move to `tests/unit/core/helpers` because there are other places that could benefit from a more realistic response. I'm thinking of saga tests and anywhere that we resolve API mocks with a response.